### PR TITLE
feat: :see_no_evil: Created the fetch from api to get manga Data

### PR DESCRIPTION
**Pull request by CharelMarais**
Created the api calls with interfaces to get manga information with cover photos

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 function App() {
-
-
   return (
-    <Loading></Loading>
-  )
+    <QueryClientProvider client={queryClient}>
+      <MangaCardContainer />
+    </QueryClientProvider>
+    // <Loading></Loading>
```
Comment (MihlaliMgwebi) [inline src/App.tsx:12]: remove dead code

**Review by sechaba-mache — APPROVED** [review]
Looks good

File: src/services/apicalls.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+export function getCoverById(id: string): Promise<CoverData> {
+  const response: Promise<CoverData> = fetch(
+    `https://api.mangadex.org/cover/${id}`
+  )
+    .then((res) => res.json())
+    .then((res) => res.data)
+    .catch((error) => {
+      console.log(error);
```
Comment (MihlaliMgwebi) [inline src/services/apicalls.ts:21]: remove console.log

**Review by MihlaliMgwebi — CHANGES_REQUESTED** [review]
Looks good. Just take a look at my comments then merge.

**Pull request merged**